src/source/PeerConnection/Rtcp.c

```diff
@@ -1,25 +1,81 @@
 #define LOG_CLASS "RtcRtcp"
 
 #include "../Include_i.h"
+#include "kvsrtcp/rtcp_api.h"
+
+static RtcpPacketType_t getDetailedRtcpPacketType(uint8_t packetType, uint8_t receptionReportCount)
+{
+    RtcpPacketType_t result = RTCP_PACKET_UNKNOWN;
+
+    switch (packetType) {
+        case RTCP_PACKET_TYPE_FIR: {
+            if (receptionReportCount == 0) {
+                result = RTCP_PACKET_FIR;
+            }
+        } break;
+
+        case RTCP_PACKET_TYPE_SENDER_REPORT: {
+            result = RTCP_PACKET_SENDER_REPORT;
+        } break;
+
+        case RTCP_PACKET_TYPE_RECEIVER_REPORT: {
+            result = RTCP_PACKET_RECEIVER_REPORT;
+        } break;
```

Comment:
curly braces around cases are unnecessary and not consistent in this PR (see below) can we remove for readability?

Reply:
They were added when i ran clangd formatting locally. Yeah i think they can be removed. Will update it.